mmdet/models/layers/transformer/utils.py

```diff
@@ -46,6 +46,73 @@ def nchw_to_nlc(x):
     return x.flatten(2).transpose(1, 2).contiguous()
 
 
+def convert_coordinate_to_encoding(coord_tensor: Tensor,
```

Comment:
Why do these two functions still exist in this pr? They should have been merged in Conditional DETR